src/libopensc/card-epass2003.c

```diff
@@ -3050,7 +3050,7 @@ epass2003_erase_card(struct sc_card *card)
 		0x31,0x32,0x33,0x34, 0x35,0x36,0x37,0x38,
 		0x31,0x32,0x33,0x34, 0x35,0x36,0x37,0x38,
 	};
-	static const unsigned char magic_pin[16] = "1234567812345678";
+	static const unsigned char magic_pin[17] = "1234567812345678";
```

Comment:
I do not think this is correct. The APDU will likely not expect the trailing null byte so either it needs to be stripped from the sizeof(magic_pin) or it needs to be defined explicitly without trailing null byte , similarly as the `install_magic_pin` variable above.

Reply:
Thanks, I missed there is `sizeof` instead of `strlen` for getting the length, I will fixed that.